### PR TITLE
Ability to run RAPID with different k/x for each huc_8

### PR DESCRIPTION
**Pull request by galvare2**

### CODE REVIEW

File: src/rrr_riv_tot_scl_prm.py [hunk 1]
```diff
@@ -88,21 +89,25 @@
 #kfac file
 #-------------------------------------------------------------------------------
 ZV_kfac=[]
+huc_8_list_k=[]
 with open(rrr_kfc_file,'rb') as csvfile:
      csvreader=csv.reader(csvfile)
      for row in csvreader:
-          ZV_kfac.append(float(row[0]))
+          huc_8_list_k.append(row[0])
```
Comment (kjfries) [inline src/rrr_riv_tot_scl_prm.py:96]: is this the only place where rrr_kfc_file gets used? just want to make sure rapid doesn't have issues with adding another entry to the row

**Review by kjfries — COMMENTED** [review]
looks good to me

File: src/rrr_riv_tot_scl_prm.py (hunk 1 above)
Comment (galvare2) [reply src/rrr_riv_tot_scl_prm.py:96]: Yes it is, at least in our pipeline.

File: src/rrr_riv_tot_scl_prm.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 #*******************************************************************************
 import sys
 import csv
+import json
 
 
 #*******************************************************************************
```
Comment (hunicole) [inline src/rrr_riv_tot_scl_prm.py:26]: The description of the declaration of variables for arg 3 and arg 4 need to be updated to huc_8_map_k and huc_8_map_x

**Review by hunicole — APPROVED** [review]
LGTM with one small comment change

**Pull request merged**